### ISSUE TITLE
Section 3.6 should more clearly connect policies and practices with standards and criteria of ICT accessibility.

### ISSUE THREAD

**Issue by jasonjgw** [w3c/wcag-maturity-model#49]
Section 3.6 describes practices which can support the procurement of ICT systems that are broadly accessible to users with disabilities. However, the success of these practices depends on having standards (including, but not limited to, international technical standards) and criteria of adequacy related to accessibility which the acquired systems are expected to meet.

That is, the organization should specify appropriate standards and criteria (including international technical standards from the W3C, ISO, or elsewhere, as applicable to the project) as the basis of its procurement contracts and evaluation of prospective suppliers. Without a well defined list of concrete requirements, it is difficult to judge success or to hold vendors accountable.

The proof points and ratings in this section should more clearly connect procurement-related activities with the organization's chosen accessibility standards and criteria of adequacy. For example, the first point in section 3.6.1.1 states that there should be an accessibility policy, but not what such a policy should specify. Later points should refer back to the standards and criteria specified in the policy.

Compare the approach taken in ISO/IEC 30071-1:2019, which is much clearer about the role of an accessibility policy and in relating the organization's development and procurement activities to the prescriptions of the policy.

**Comment by Helixopp** [comment]
Thank you for your suggestion. Our response is as follows: Per the document Abstract, a maturity model is not a specification, and by design, technology neutral with regard to standards. It would be the responsibility of the organization to identify, and ensure, that the proper specifications are included in procurement related documents and processes.The action of doing so advances the organiation on the Maturity Model.
